pysteps/downscaling/rainfarm.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+"""
+pysteps.downscaling.rainfarm
+============================
+
+Implementation of the RainFARM stochastic downscaling method as described in
+:cite:`Rebora2006`.
+
+.. autosummary::
+    :toctree: ../generated/
+
+    downscale
+"""
+
+import warnings
+
+import numpy as np
+from scipy.ndimage import convolve
+
+
+def _log_slope(log_k, log_power_spectrum):
+    lk_min = log_k.min()
+    lk_max = log_k.max()
+    lk_range = lk_max - lk_min
+    lk_min += (1 / 6) * lk_range
+    lk_max -= (1 / 6) * lk_range
+    dk = lk_range / 32
+
+    selected = (lk_min <= log_k) & (log_k <= lk_max)
+    num_selected = np.count_nonzero(selected)
+    lk_sel = log_k[selected]
+    ps_sel = log_power_spectrum[selected]
+    (alpha, _) = np.polyfit(lk_sel, ps_sel, 1)
```

Comment:
`(alpha, _) = np.polyfit`.  suggests that `polyfit` returns a tuple of 2 different arguments while it actually returns a single array (of 2 elements).  
```suggestion
    alpha = np.polyfit(lk_sel, ps_sel, 1)[0]
```